docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+# Telegraf autopilotpattern demo
+
+# Consul - start with a single host which will bootstrap the cluster.
+# In production we'll want to use an HA cluster.
+consul:
+    container_name: consul
+    image: progrium/consul:latest
```

Comment:
Any reason to run `progrium/consul` over the [official consul image](https://hub.docker.com/_/consul/)?